### PR TITLE
[license] Add header to test java files

### PR DESCRIPTION
**Pull request by JingsongLi**
### Purpose

Even java files in resources, we need to add header to these files.

From https://lists.apache.org/thread/2fdsps5ggllo56oy61cp0gwb8spov1vf

- These files are missing ASF headers [3][4][5] and other files in that directory, and these directories [6][7][8][9][10] and other directories under [11]. Are they ASF files and how are they licensed?

### CODE REVIEW

**Review by carp84 — CHANGES_REQUESTED** [review]
It seems to me that in our [apache-rat plugin configuration](https://github.com/apache/incubator-paimon/blob/master/pom.xml#L411) (in the pom file) we have excluded all codesplit resources, where we are missing license headers and trying to fix in this PR. I suggest we carefully review our apache-rat configuration and do necessary change to make sure no more unexpected exclusions.

Note: files such as `paimon-common/src/test/resources/codesplit/groups` should be excluded indeed, and we need to carefully check and make sure that `mvn verify` has the precise coverage (no more no less).

File: pom.xml
Comment (carp84) [inline pom.xml:1]: The current changes LGTM, and I have confirmed `mvn verify` could pass.

Could you further make sure whether the below exclusions in the rat check configuration are as expected and already with necessary notice/license declaration?:
1. docs/assets/github.css (L399)
2. docs/static/js/anchor.min.js (L400)
3. docs/layouts/shortcodes/generated/** (L401)

And please make sure the exclusion of font-awesome (L395) is removed in PR#1192 if the dependency is removed there. Thanks.

Comment (JingsongLi) [reply pom.xml:1]: >1. docs/assets/github.css (L399)

Add license in https://github.com/apache/incubator-paimon/pull/1192

>2.docs/static/js/anchor.min.js (L400)

Already noticed in NOTICE and LICENSE

>3.docs/layouts/shortcodes/generated/** (L401)

Addressed, I generated in `paimon-doc`.

Very thanks Yu.

**Review by carp84 — APPROVED** [review]
The latest changes LGTM, and I've confirmed `mvn verify` could pass locally. Thanks for the efforts @JingsongLi

**Pull request merged**